service/key/key.go

```diff
@@ -17,6 +17,9 @@ const (
 	monitoring = "monitoring"
 
 	PrometheusMemoryLimitCoefficient float64 = 1.2
+	PrometheusServiceName                    = "prometheus-operated"
+	RemoteWriteAgentConfigSecretName string  = "agent-remote-write-config"
+	RemoteWriteAgentSecretName       string  = "agent-remote-write"
```

Comment:
In general I would prefer to avoid being too specific. We are just configuring a remote write endpoint here. We do not need to mention the `agent` thing here. It could be consumed by a standard Prometheus also, and this might changes. Let's avoid being too specific please. This is also valid for package names.

Reply:
Except we are not just configuring a remote write endpoint, we are configuring it for the agent. We already have remotewritesecret and remotewriteconfig as resources anyway so we need to avoid collision between the 2

Reply:
Maybe something like `incoming-remote-write` ? `agent` feels wrong here, this is just a mode used to run Prometheus.